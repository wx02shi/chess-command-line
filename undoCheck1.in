game human human
setup
+ K a1
= black
+ K h7
+ Q b7
done
move a1 b1
