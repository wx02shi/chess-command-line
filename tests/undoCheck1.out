8 rnbqkbnr
7 pppppppp
6  _ _ _ _
5 _ _ _ _ 
4  _ _ _ _
3 _ _ _ _ 
2 PPPPPPPP
1 RNBQKBNR

  abcdefgh
8  _ _ _ _
7 _ _ _ _ 
6  _ _ _ _
5 _ _ _ _ 
4  _ _ _ _
3 _ _ _ _ 
2  _ _ _ _
1 K _ _ _ 

  abcdefgh
8  _ _ _ _
7 _ _ _ _k
6  _ _ _ _
5 _ _ _ _ 
4  _ _ _ _
3 _ _ _ _ 
2  _ _ _ _
1 K _ _ _ 

  abcdefgh
8  _ _ _ _
7 _q_ _ _k
6  _ _ _ _
5 _ _ _ _ 
4  _ _ _ _
3 _ _ _ _ 
2  _ _ _ _
1 K _ _ _ 

  abcdefgh
Setup completed!
Cannot move to cause a self-check.
Final Score:
White: 0
Black: 0
